File: Matlab/int-ai4.m
% Define the function to be integrated
f = @(x) abs(sin(x));

% Set the limits of integration
a = -pi;
b = 2*pi/3;

% Perform the integration using the integral function
area = integral(f, a, b);

% Display the result
fprintf('The total area between sin(x) and the x-axis from -pi to 2/3*pi is: %.4f\n', area);

File: R/int-ai4.r
# Define the function to integrate
f <- function(x) abs(sin(x))

# Set the limits of integration
lower_limit <- -pi
upper_limit <- 2/3 * pi

# Perform the integration
area_result <- integrate(f, lower_limit, upper_limit)

# Print the result
print(paste("The total area between sin(x) and the x-axis from -pi to 2/3*pi is:", area_result$value))
